fdk_rdf_parser_service/rabbit/parser_ingest.py

```diff
@@ -1,16 +1,27 @@
 """Ingester for messages."""
+import json
 import logging
 
+from fdk_rdf_parser_service.model.rabbit_report import RabbitReport
 
-def ingest_for_index(index_key: str) -> None:
+
+def ingest_for_index(index_key: str, body: bytes) -> None:
```

Comment:
Hva betyr ingest for index?

Reply:
Hentet fra fdk-fulltext-search-rabbit-consumer, men index gir jo ikke noe mening i den sammenhengen her. Kan endre:)

Reply:
Så den konsumerer rabbit-meldingene per katalogtype (index='dataset', 'concept', 'informationmodel', osv)

Reply:
sånn endret:)